docs/usage/administration/monthly_active_users.md

```diff
@@ -0,0 +1,73 @@
+# Monthly Active Users
+
+Synapse can be configured to record the number of monthly active users (also referred to as MAU) on a given homeserver.
+For clarity's sake, MAU only tracks local users.
+
+Please note that the metrics recorded by the [Homeserver Usage Stats](/usage/administration/monitoriing/reporting_homeserver_usage_statistics)
+are calculated differently. The `monthly_active_users` does not take into account any of the rules below, and counts any users 
+who have made a request to the homeserver in the last 30 days.
+
+See the [configuration manual](/usage/configuration/config_documentation.html#limit_usage_by_mau) for details on how to configure MAU.
+
+## Calculating active users
+
+Individual user activity is measured in active days, so if a user performs an action on a given day, that day is then recorded. When
+calculating the MAU figure, any users with a recorded action in the last 30 days are considered part of the cohort.
```

Comment:
Not to get into weird timezone things but...what's "a day" and what's "30 days" -- from below it sounds like the 30 day window is rolling? (So it is `now - 30 days`.)

Reply:
Good point, I had that assumption in my head but failed to write it. Have tried to clarify, but unsure yet if it's made it better or worse.